task/01-strings-tasks.js

```diff
@@ -69,6 +75,7 @@ function getStringFromTemplate(firstName, lastName) {
  *   'Hello, Chuck Norris!' => 'Chuck Norris'
  */
 function extractNameFromTemplate(value) {
+    return value.slice(7, value.length-1);
```

Comment:
looks good, but please note slice can work with negative numbers also to take symbols from end. In the case `slice(7, -1);` can do the same.